src/datasets/data_files.py

```diff
@@ -393,7 +422,7 @@ def resolve_patterns_in_dataset_repository(
     return data_files_urls
 
 
-def get_patterns_in_dataset_repository(
+def get_data_patterns_in_dataset_repository(
```

Comment:
This is a breaking change. 

See in:
- #4610 

There might be Hub dataset scripts that use `get_patterns_in_dataset_repository`.

CC: @mariosasko @lhoestq